packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -296,14 +279,32 @@ export const replaceReactComponentsImports = (source: string): boolean => {
 
 /**
  * Removes old fetch import that is no longer used
- * @param source
+ * @param source the src folder
  */
 export const removeFetchImport = (source: string) => {
   const operation = async (filePath: string) => {
+    let hasReplaced = false;
     const fileContent = fs.readFileSync(filePath, "utf8");
-    if (fileContent.match(fetchImportRegex)) {
-      const modifiedContent = fileContent.replace(fetchImportRegex, "");
-      fs.writeFileSync(filePath, modifiedContent, "utf8"); // Update the file content
+    let lines = fileContent.split("\n");
+    for (let i = 0; i < lines.length; i++) {
```

Comment:
Seems pretty inefficient to loop through every line of every file. Is there a better way we can get just the import lines, either through a regex or ts-morph?

Reply:
You could also stop search as soon as you find a line that doesn't have `import`